src/OnChain.ts

```diff
@@ -87,7 +89,7 @@ export const OnChainMixin = (superclass) =>
       if (payeeUser) {
         fee = 0
       } else {
-        if (amount && amount < yamlConfig.onchainDustAmount) {
+        if (amount && amount < this.config.dustThreshold) {
```

Comment:
Shouldn't this be `onChainWallet.dustThreshold`?

Reply:
I suggested to have the object specific config flattened. I like this because it reduces the amount of structural information that must be known at the point of usage. This also decouples the object specific config from the mechanism with by which the global config is provisioned. Achieving this decoupling was the primary goal of this refactoring.